nixos/doc/manual/from_md/release-notes/rl-2211.section.xml

```diff
@@ -139,6 +139,14 @@
           <link xlink:href="options.html#opt-services.hadoop.hbase.enable">services.hadoop.hbase</link>.
         </para>
       </listitem>
+      <listitem>
```

Comment:
I don't think you need to add both the markdown and the xml version of this documentation. Remove this, and add only a markdown line in the latest `rl-2311.section.md` file.